src/js/ia/IADevPipeline.js

```diff
@@ -144,6 +144,30 @@
                 }
             });
 
+            $("body").on("click", "#beta_install", function(evt) {
+                var prs = [];
+                $(".dev_pipeline-column__list .selected").each(function(idx) {
+                    var temp_pr = $.trim($(this).find(".item-activity a").attr("href"));
+                    var temp_pr_id = temp_pr.replace(/.*\//, "");
+                    var temp_repo = temp_pr.replace(/^.*\/duckduckgo\//, "").replace(/\/[a-zA-Z0-9]*\/[a-zA-Z0-9]*\/?/, "");
+
+                    if (temp_pr_id && temp_repo) {
+                        var temp_hash = 
+                            {
+                                "action" : "install",
+                                "number" : temp_pr_id,
+                                "repo" : temp_repo
+                            };
+
+                        prs.push(temp_hash);
+                    }
+                });
+
+                if (prs.length) {
+                    send_to_beta(JSON.stringify(prs[0]));
```

Comment:
You can send these all in a loop now.  Not ideal but I got it working on the comserver :smile: